src/main/java/sebastian/main/Storage.java

```diff
@@ -0,0 +1,87 @@
+package sebastian.main;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import sebastian.exceptions.CannotLoadDataException;
+import sebastian.exceptions.CannotWriteDataException;
+import sebastian.exceptions.DeadlineFormatMismatchException;
+import sebastian.exceptions.EventFormatMismatchException;
+import sebastian.task.TaskStatus;
+
+
+/**
+ * A class to store task list to hard disk and to retrieve data from the hard disk
+ */
+public class Storage {
+    private File file;
+
+    public Storage(String filePath) {
+        this.file = new File(filePath);
+    }
+
+    /**
+     * Load data from hard disk and convert the String representation into actual tasks.
+     * @return a main.TaskList containing all the tasks.
+     */
+    public TaskList formTaskListFromData() throws CannotLoadDataException {
+        TaskList taskList = new TaskList();
+        File file = new File("src/main/resources/SebastianData.txt");
```

Comment:
Perhaps it might be better to extract out the file path for your Storage class methods? Your Storage class constructor receives a String _filePath_ as argument and creates a _file_ field but it is immediately modified in your methods. It is also more convenient if you decide to change your file path in the future.